lib/bn.js

```diff
@@ -731,6 +731,45 @@ BN.prototype.uxor = function uxor(num) {
 };
 
 
+// Not ``this`` with ``width`` bitwidth
+BN.prototype.inotn = function inotn(width) {
+  assert(typeof width === 'number' && width >= 0);
+
+  var bytesNeeded = width / 26;
```

Comment:
It doesn't really make a difference - even with 1/26 it is 0.0384.. and that is bigger than 0. Added it back for broken JS interpreters which would evaluate 0<0.0384 as false.